features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -158,3 +147,31 @@ Feature: Creating a catalog promotion
         And I add a new catalog promotion action
         Then I should see the catalog promotion scope configuration form
         And I should see the catalog promotion action configuration form
+
+    @api @ui @javascript
+    Scenario: Adding a new catalog promotion that applies on variant with exclusive promotion that has lower priority already applied
+        When there is an exclusive catalog promotion "Spring sale" with priority 120 that reduces price by "50%" and applies on "Kotlin T-Shirt" variant
+        And there is a catalog promotion "PHP stuff promotion" with priority 100 that reduces price by fixed "$10" in the "United States" channel and applies on "T-Shirt" product
```

Comment:
These are the setup steps that should be in `Given` part, not in `When` as in these steps, the catalog promotion is created programmatically, not by an API or in UI. The same problem is in other scenarios.